src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionWorkflowRepository.java

```diff
@@ -22,4 +22,17 @@ WHERE ua.userFullName ILIKE CONCAT('%', :owner, '%')
             GROUP BY tw.transcription.id
             """)
     List<TranscriptionIdsAndLatestWorkflowTs> findWorkflowOwnedBy(String owner);
-}
+
+    @Query("""
+            SELECT distinct tw.transcription
+            FROM TranscriptionWorkflowEntity tw
+            JOIN tw.transcription trans
+            JOIN tw.workflowActor ua
+            WHERE trans.transcriptionStatus.id = :statusId
+            AND ua.id = :userId
+            """)
+    List<TranscriptionEntity> findWorkflowForUserWithTranscriptionState(
```

Comment:
for complex queries like this one would be good to have an integration test on the repository method in isolation